unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/HUDController.cs

```diff
@@ -329,6 +331,9 @@ public void ConfigureHUDElement(HUDElementID hudElementId, HUDConfiguration conf
             case HUDElementID.AVATAR_NAMES:
                 // TODO Remove the HUDElementId once kernel stops sending the Configure HUD message
                 break;
+            case HUDElementID.TRANSACTIONS:
+                CreateHudElement(configuration, hudElementId);
```

Comment:
The HUDController is deprecated, we are moving away from it. We should try to avoid creating new elements on it. 

I think the Plugin system will be the correct way to implement it but I don't know the implementation details. Maybe @BrianAmadori  can give more insights here